target/linux/rockchip/base-files/lib/preinit/79_move_config

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
I'm not sure whether preinit files are executed or sourced ...

Reply:
They are sourced indeed (see /etc/preinit), but every preinit file on my OpenWRT installation has a shebang, so I think I better leave this in?

Reply:
Well, most people just don't care. But I personally prefer to have this tidy, just like you wouldn't give the file an executable bit without need.

Reply:
OK, I've changed it now :)